deploy/kubernetes/helm/che/templates/dashboard-traefik-gateway-configmap.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+{{- if and (eq .Values.global.serverStrategy "single-host") (eq .Values.global.singleHostExposure "gateway") }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: che
+    component: che-gateway-config
+  name: che-gateway-route-dashboard
+data:
+  dashboard.yml: |
+    http:
+      routers:
+        che-dashboard:
+          rule: "PathPrefix(`/dashboard/`)"
```

Comment:
does it work without end slash?

Reply:
Checking, I assume that if we miss end slash, then we'll get broken redirects, like /dashboardanythings -> /dashboard since it's currently done by Che Server.

Reply:
![Screenshot_20201013_103025](https://user-images.githubusercontent.com/5887312/95829615-203e8100-0d3f-11eb-9dd7-53a95a8445a2.png)

First Dashboard ingress with `/`, the second without. Do you see any issue with end slash?

Reply:
Added end slash to ingress http path as well.